### PR TITLE
JDG-2303 Merge the datagrid standalone and openshift images

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.jboss.org/browse/JDG-2303

@galderz @osnipso This is pretty much a straight copy of the module from the https://github.com/jboss-container-images/jboss-datagrid-7-image repo, the only interesting thing is that I had to temporarily unset the jboss modules env during the patch process https://github.com/ryanemerson/jboss-datagrid-7-openshift-image/blob/ccf03e6b5a9819203db4cc17f5d628cc8f35b38b/modules/datagrid/72/distribution/install_patches.sh#L65-L67.

### CODE REVIEW

File: image.yaml
Comment (osnipso) [inline image.yaml:6]: This change to use the OpenJDK base image reminds me: CE is not going to build base images anymore in the future. We'll need to skip both the openjdk18-rhel7 and base-rhel7 layers, basing our images directly in rhel7 and incorporating what we need from those two images through modules (composition), not inheritance.
Do you prefer to integrate this change and have a build for this unified image before tackling that, or do you think we could take a look into doing all those changes at once?

**Review by galderz — APPROVED** [review]

**Comment by galderz** [comment]
Looks good to me, @osnipso can you also have a look?

**Review by osnipso — APPROVED** [review]
@galderz I had already added a comment. Anyway, the changes look good to me, I would just like to discuss in the near future about basing the image directly off of rhel7, because of the phasing out of base-rhel7 and openjdk18-rhel7.
~@ryanemerson, could you please rebase and update the PR, so we don't start a new line in the repo? I'll merge right after it. Thanks.~ <<< Just have hit rebase and merge, thanks.

File: image.yaml
Comment (ryanemerson) [reply image.yaml:6]: @osnipso Thanks for merging. I think that the work related to basing our images directly on rhel7 is definitely a separate issue. Best to do things in stages and keep the PRs as simple as possible.

**Pull request merged**
